index.html

```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
+            </div>
+            <div class="first-project">
+              <h3>
+                Setup-Portfolio-Mobile-1st <br>
+                First Week
+              </h3>
+                <div class="languages-grid">                    
+                        <div class="lang-list l1"><strong>Html5</strong></div>
+                        <div class="lang-list l2"><strong>Css3</strong></div>
+                        <div class="lang-list l3"><strong>Git</strong></div>
+                        <div class="lang-list l4"><strong>Github</strong></div>
+                </div>
+                <div class="external_link">
+                    <a href="#">See Project</a>
+                </div>
+            </div>
+        </div>       
+    </div>
+    <div id="about-me">
+        <div class="about-me-bg">
+        <h1 class="about-me-tittle">About me</h1>
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</p>
+        <div class="external_link" id="ex-li-3">
+            <a href="#">Get my Resume</a>
+        </div>
+        </div>
+        <div class="project-1" id="about-me-section-1">
+            <div class="am-languages-main-grid">
+                
+                    <img src="img/Icon Langguage.png" alt="first project image" class="icon_language">
+                    
+              <h3 id="h3-lang">
+                Languages
+              </h3>
+                <div class="am-languages-grid">                    
+                        <div class="am-lang-list am1"><strong>Html5</strong></div>
+                        <div class="am-lang-list am2"><strong>Css3</strong></div>
+                        <div class="am-lang-list am3"><strong>Git</strong></div>
+                        <div class="am-lang-list am4"><strong>Github</strong></div>
+                </div>
+            </div>
+        </div>
+            <div class="project-1" id="about-me-section-2">
+                <div class="am-frameworks-main-grid">
+                    
+                        <img src="img/Icon Framework.png" alt="first project image" class="icon_language">
+                        
+                  <h3 id="h3-lang">
+                    Frameworks
+                  </h3>
+                    <div class="am-frameworks-grid">                    
+                            <div class="am-frame-list f1"><strong>Angular</strong></div>
+                            <div class="am-frame-list f2"><strong>React</strong></div>
+                            <div class="am-frame-list f3"><strong>Spring</strong></div>
+                            <div class="am-frame-list f4"><strong>ASP.NET</strong></div>
+                            <div class="am-frame-list f5"><strong>NodeJS</strong></div>
```

Comment:
- [ ] Please refer to line `63` on the use of proper semantics for this section 👍